_posts/0000-01-12-Gymky.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Changing caption of my cool octocat
 
 ![cooler-octocat](https://octodex.github.com/images/twenty-percent-cooler-octocat.png)
```

Comment:
If you don't like your image, you can always pick a new one from [here](https://octodex.github.com/)

Reply:
image was good, just didn't know what to name it.